backend/main.py

```diff
@@ -85,12 +93,15 @@ def resource_not_found_exception_handler(
 
 # Add feature-specific exception handling middleware
 from .api import coworking
+from .api import events
 
-feature_exception_handlers = [coworking.exception_handlers]
+feature_exception_handlers = [coworking.exception_handlers, events.exception_handlers]
 
 for feature_exception_handler in feature_exception_handlers:
     for exception, handler in feature_exception_handler:
 
         @app.exception_handler(exception)
-        def _handler_wrapper(request: Request, e: exception):
+        def _handler_wrapper(
+            request: Request, e: ReservationException | EventRegistrationException
+        ):
```

Comment:
I'm going to push a minor fix here that uses `Exception` as the common supertype rather than unioning the subtypes. Not sure why it was ever lowercase `exception`, but that's almost certainly on me 🤡

Reply:
Pushed this fix.